### PR TITLE
ENG21-858 Adding simple check icon

### PR DESCRIPTION
**Pull request by mcadizwtw**
Adding simple check icon:

image

### CODE REVIEW

**Review by martinmunillas — APPROVED** [review]

**Review by ARivas16 — CHANGES_REQUESTED** [review]
The same comment as Cris

**Comment by ARivas16** [comment]
and maybe the name of the component should be more descriptive to identify why this is different from the existing CCheck component

**Review by crgp01 — APPROVED** [review]

**Comment by mcadizwtw** [comment]
@ARivas16 as we talked, I don't think that this component should have a more descriptive name, since it's just a Check icon, with nothing added. Most of the icons libraries use by convention to have a base icon like `CheckIcon` and then have variants like `CircledCheckIcon` or `SquaredCheckIcon` and things like that. And for this case, this is kind of the base icon for others like `CCheck`

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "wtw-icons",
-    "version": "1.38.0",
+    "version": "1.38.1",
```
Comment (ARivas16) [inline package.json:3]: I know you are just adding a new icon but in theory, according to semantic versioning, this isn't a fix https://semver.org/ it's a new icon... So, it should be a new version non-breaking like `1.39.0`

**Review by ARivas16 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (mcadizwtw) [reply package.json:3]: You are right

**Review by ARivas16 — APPROVED** [review]

**Pull request merged**